demo/actions/incident_commands.py

```diff
@@ -0,0 +1,7 @@
+from response.core.models import Incident
+from response.slack.decorators.incident_command import incident_command
+
+#remove this before committing as its testing
```

Comment:
Should this have been removed after testing it was importing the custom actions now?

Reply:
Yup, removed it as we did testing - will add this back in another PR with real features.